introduction.md

```diff
@@ -0,0 +1 @@
+My name is Marcin Sieka. I work for Hewlett-Packard as a Programmer. I am 32 years old. 
```

Comment:
Looks good! Could you also please mention your favorite Code School path in your introduction? This could be Ruby, JavaScript, HTML & CSS, iOS or Electives.